src/test/java/guitests/LogoutTest.java

```diff
@@ -31,11 +32,11 @@ public void logoutFunctionTest() {
         type("dummy").push(KeyCode.TAB);
         type("test").push(KeyCode.TAB);
         type("test");
-        click("Sign in");
+        clickOn("Sign in");
         sleep(2000);
-
-        click("File");
-        click("Logout");
+        
```

Comment:
Trailing whitespace. You can set eclipse's save action to remove all trailing white spaces since we might not be able to detect all of those manually.